public/components/FeatureAnywhereContextMenu/AssociatedMonitors/AssociatedMonitors.js

```diff
@@ -0,0 +1,94 @@
+import React, { useCallback } from 'react';
+import {
+  EuiFlyoutHeader,
+  EuiTitle,
+  EuiText,
+  EuiSpacer,
+  EuiInMemoryTable,
+  EuiFlyoutBody,
+  EuiEmptyPrompt,
+  EuiButton,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiLoadingSpinner,
+  EuiTextColor,
+} from '@elastic/eui';
+import './styles.scss';
+import { useColumns } from './helpers';
+
+const AssociatedMonitors = ({ embeddable, closeFlyout, setMode, monitors }) => {
+  const title = embeddable.getTitle();
+  const onUnlink = useCallback(
+    (item) => {
+      console.log('onUnlink', item);
+      closeFlyout();
+    },
+    [closeFlyout]
+  );
+  const onEdit = useCallback(
+    (item) => {
+      console.log('onEdit', item);
+      closeFlyout();
+    },
+    [closeFlyout]
+  );
+  const columns = useColumns({ onUnlink, onEdit });
+  const empty = (
+    <EuiEmptyPrompt
+      title={<h3>No monitors to display</h3>}
+      titleSize="s"
+      body={`There are no alerting monitors associated with ${title} visualization.`}
+    />
+  );
+  const loading = <EuiEmptyPrompt body={<EuiLoadingSpinner size="l" />} />;
+  const tableProps = {
+    items: monitors || [],
+    columns,
+    search: {
+      box: {
+        disabled: !monitors || monitors.length === 0,
+        incremental: true,
+        schema: true,
+      },
+    },
+    hasActions: true,
+    pagination: true,
+    sorting: true,
+    message: monitors ? empty : loading,
+  };
+
+  return (
+    <div className="associated-monitors">
```

Comment:
This should be `EuiFlyout` instead of `div`?

Reply:
Correct, there is a container Flyout that this component is launched within. In the `utils/contextMenu/getActions.tsx` you can see where the flyout is created.